### PR TITLE
upgrade to pion/webrtc v4

### PR DESCRIPTION
**Pull request by cnderrauber**

### CODE REVIEW

**Review by paulwe — APPROVED** [review]

**Review by boks1971 — APPROVED** [review]
🚀

File: go.mod [hunk 1]
```diff
@@ -4,41 +4,45 @@ go 1.18
...
+	github.com/pion/transport/v3 v3.0.7
+	github.com/pion/webrtc/v4 v4.0.3
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.8.4
+	github.com/stretchr/testify v1.10.0
 	golang.org/x/exp v0.0.0-20230626212559-97b1e661b5df
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
```
Comment (paulwe) [inline go.mod:21]: nit: can this be merged with the require block above?

Comment (cnderrauber) [reply go.mod:21]: it is format by `go mod tidy`, do we need to merge it manually?

Comment (paulwe) [reply go.mod:21]: yeah, you merge the blocks manually then re-run `go mod tidy` and it will sort them for you

**Pull request merged**